crates/wasmcloud-host/src/host.rs

```diff
@@ -521,6 +521,47 @@ impl Host {
             .collect())
     }
 
+    /// Retrieves human-friendly identity information for a provider
+    /// Return value is `(image_reference, name, revision)`
```

Comment:
Should be explicit here that the `id` parameter is the public key.

Reply:
Good call. I'll rename it to `actor_id` as well to make it immediately clear